logbook-bom/pom.xml

```diff
@@ -20,30 +20,37 @@
             <dependency>
                 <groupId>org.zalando</groupId>
                 <artifactId>logbook-api</artifactId>
+                <version>1.12.0-SNAPSHOT</version>
```

Comment:
Is it possible to use a property pointing to it's own version? Otherwise we need to make sure to bump this here: 

https://github.com/zalando/logbook/blob/936deddc73ef6e82908a5db137480cbd6f952ee8/release.sh#L11